src/containerapp/azext_containerapp/_utils.py

```diff
@@ -161,10 +221,9 @@ def parse_service_bindings(cmd, service_bindings_list, resource_group_name, name
         service_binding = parts[0].split(':')
         service_name = service_binding[0]
 
-        if len(service_binding) == 1:
-            binding_name = service_name
-        else:
-            binding_name = service_binding[1]
+        is_kafka = service_name == "kafkaconfluent"
```

Comment:
Recomments make `kafkaconfluent` a constant value.

Reply:
i can. will do. this will change as soon as the Kafka on Confluent RP fully onboard to Azure but it can be a constant for now.